README.adoc

```diff
@@ -279,6 +279,11 @@ The base command is different whether you choose guice flavor or spring :
 
 DOMAIN: is the domain you want to add.
 
+Note: Using docker, one can add an environment variable holding the domain to be created. This
```

Comment:
And using the inneficient retrieve hostname DNS feature?

That is the first thing @rouazana  turned off on OP.LNG.COM, I still support this feature, on a ticket you created and defended. Let's do it the conventional docker way, and IMO let's not rely on a hack.